--- ./.github/workflows/ci.yml	2024-11-29 15:27:49
+++ ./.github/workflows/ci.yml~	2024-11-29 15:27:40
@@ -16,7 +16,7 @@
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: .ruby-version
+          ruby-version: 3.3
           bundler-cache: true
 
       - name: Scan for common Rails security vulnerabilities using static analysis
@@ -32,7 +32,7 @@
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: .ruby-version
+          ruby-version: 3.3
           bundler-cache: true
 
       - name: Scan for security vulnerabilities in JavaScript dependencies
@@ -47,9 +47,8 @@
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: .ruby-version
+          ruby-version: 3.3
           bundler-cache: true
 
       - name: Lint code for consistent style
         run: bin/rubocop -f github
-
