--- config/database.yml	2024-11-28 21:13:48
+++ config/database.yml~	2024-11-28 21:19:29
@@ -22,25 +22,26 @@
 
 development:
   <<: *default
-  database: {{cookiecutter.project_slug.replace('-', '_')}}_development
+  # plate_spinner_rails_development
+  database: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_DEV_DATABASE_DATABASE'] %>
 
   # The specified database role being used to connect to PostgreSQL.
   # To create additional roles in PostgreSQL see `$ createuser --help`.
   # When left blank, PostgreSQL will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: {{cookiecutter.project_slug.replace('-', '_')}}
+  username: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_DEV_DATABASE_USERNAME'] %>
 
   # The password associated with the PostgreSQL role (username).
-  #password:
+  password: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_DEV_DATABASE_PASSWORD'] %>
 
   # Connect on a TCP socket. Omitted by default since the client uses a
   # domain socket that doesn't need configuration. Windows does not have
   # domain sockets, so uncomment these lines.
-  #host: localhost
+  host: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_DEV_DATABASE_SERVER'] %>
 
   # The TCP port the server listens on. Defaults to 5432.
   # If your server runs on a different port number, change accordingly.
-  #port: 5432
+  port: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_DEV_DATABASE_PORT'] %>
 
   # Schema search path. The server defaults to $user,public
   #schema_search_path: myapp,sharedapp,public
@@ -56,7 +57,13 @@
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: {{cookiecutter.project_slug.replace('-', '_')}}_test
+  database: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_TEST_DATABASE_DATABASE'] %>
+  username: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_TEST_DATABASE_USERNAME'] %>
+  password: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_TEST_DATABASE_PASSWORD'] %>
+  # don't use unix domain socket, use TCP/IP, so we can host database via
+  # Docker:
+  host: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_TEST_DATABASE_SERVER'] %>
+  port: <%= ENV['{{cookiecutter.project_slug.replace('-', '_').upper()}}_TEST_DATABASE_PORT'] %>
 
 # As with config/credentials.yml, you never want to store sensitive information,
 # like your database password, in your source code. If your source code is
@@ -78,7 +85,5 @@
 # for a full overview on how database connection configuration can be specified.
 #
 production:
+  url: <%= ENV.fetch('PROD_DATABASE_URL', ENV['DATABASE_URL']) %>
   <<: *default
-  database: {{cookiecutter.project_slug.replace('-', '_')}}_production
-  username: {{cookiecutter.project_slug.replace('-', '_')}}
-  password: <%= ENV["{{cookiecutter.project_slug.replace('-', '_').upper()}}_DATABASE_PASSWORD"] %>
