--- ./bin/bundle	2025-07-07 08:16:37
+++ ./bin/bundle~	2025-07-07 08:16:18
@@ -39,6 +39,7 @@

   def gemfile
     gemfile = ENV["BUNDLE_GEMFILE"]
-    return gemfile if gemfile && !gemfile.empty?
+    # https://stackoverflow.com/questions/67910193/heroku-bundler-undefined-method-present-fails-to-install-gems-via-bundler
+    return gemfile if gemfile && !gemfile.empty? # rubocop:disable Rails/Present

     File.expand_path("../Gemfile", __dir__)
   end
